docs/dasharo-menu-docs/dasharo-system-features.md

```diff
@@ -175,6 +175,16 @@ Flow chart for the two settings mentioned above:
 
 ![](../images/menus/pwr_fail_opt.png){ class="center" }
 
+- `USB ports power and charging` - Controls whether power is supplied to USB
+  ports while the platform is off. The available options are:
+    + `While System Is On` - The USB ports will be powered only while the device
+      is turned on or in sleep mode.
+    + `Always On` - The USB ports will remain powered even while the device is
+      turned off. This may cause a reduction in standby battery life in battery
+      powered devices. Availability of ports that are powered in this mode
+      depends on your specific device, check your mainboard's Hardware
+      Configuration Matrix on docs.dasharo.com for a detailed specification.
```

Comment:
This is doc.dasharo.com so no need to say it explicitly:

```suggestion
      Configuration Matrix in the relevant model documentation in [Supported Hardware section](https://docs.dasharo.com/variants/overview/) for a detailed specification.
```

Reply:
Applied: https://github.com/Dasharo/docs/pull/1009/commits/07cbc5fea717ee6fbd4207cac6c90eb1525921f1

Reply:
My bad, it probably should be relative link to `variants/overview.md`